plugins/woocommerce/client/admin/client/settings-payments/components/header/header.tsx

```diff
@@ -26,6 +28,22 @@ interface HeaderProps {
 	 * The link to go back to. If not provided, the back button will not be shown.
 	 */
 	backLink?: string;
+	/**
+	 * The description of the header.
+	 */
+	description?: string;
+	/**
+	 * Whether to show the button or not.
+	 */
+	hasButton?: boolean;
```

Comment:
I notice this param is optional. What is the default if it is not passed in?

Reply:
Since the parameter is a boolean, its default value would be false.